lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -18,6 +18,7 @@ has base_url      => '/packed/';
 has minify        => 0;
 has preprocessors => sub { Mojolicious::Plugin::AssetPack::Preprocessors->new };
 has out_dir       => '';
+has source_paths  => sub { shift->_app->static->paths };
```

Comment:
Hi Jan,

That does look like an idea, I haven't had much free time to work on 
anything the past week (busy moving from Indonesia to Europe again) so 
I'll get to it when I have time :)

On 08/05/2015 03:55 PM, Jan Henning Thorsen wrote:
>
> In lib/Mojolicious/Plugin/AssetPack.pm 
> <https://github.com/jhthorsen/mojolicious-plugin-assetpack/pull/58#discussion_r36281339>:
>
> > @@ -18,6 +18,7 @@ has base_url      => '/packed/';
> >  has minify        => 0;
> >  has preprocessors => sub { Mojolicious::Plugin::AssetPack::Preprocessors->new };
> >  has out_dir       => '';
> > +has source_paths  => sub { shift->_app->static->paths };
>
> I was wondering if it could be an idea to autodetect an "assets" 
> directory as well, or maybe "assetpack" to make it more back compat. 
> Something like this:
>
> |has source_paths  => sub {
>    my $self = shift;
>    my $default = $self->_app->home->rel_dir('assetpack');
>    return [$default] if -d $default;
>    return $self->_app->static->paths;
> };
> |
>
> See also the comment below about |$config->{source_paths}|, where this 
> can easily be done another way.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/jhthorsen/mojolicious-plugin-assetpack/pull/58/files#r36281339>.
>